.github/workflows/test.yml

```
@@ -25,7 +25,7 @@ jobs:
 
       - name: Install dependencies and app
         run: |
-          poetry install --with layer-database --with layer-python
+          poetry install --with layer-database --with layer-tools --with layer-processing
```

Maybe call this the SPICE layer?

```suggestion
          poetry install --with layer-database --with layer-spice --with layer-processing
```